### PR TITLE
Refractor mongo db

### PR DESCRIPTION
**Pull request by boldCM**
- created a help function for find.sort.toArray
- splitted code into more files instead of one big file for connection to MongoDB
- created forEach Loop for same route-structure for deleteMany fundion

### CODE REVIEW

File: client/src/api/connectJSON.js [hunk 1]
```diff
@@ -59,7 +58,7 @@ export async function deleteWholeWeek() {
 }
 
 export async function checkShoppingList() {
-  const shoppingResponse = await fetch("/api/shoppingList");
+  const shoppingResponse = await fetch("/api/shoppingItems");
   const listStatus = await shoppingResponse.json();
   if (listStatus) {
     return listStatus;
```
Comment (bwnstck) [inline client/src/api/connectJSON.js]:
```suggestion
  return listStatus ? listStatus : false
```
and do you really need the `false` in the return-statement?

File: server.js [hunk 2]
```diff
@@ -2,26 +2,41 @@ const express = require("express");
 const path = require("path");
 require("dotenv").config();
```
Comment (bwnstck) [inline server.js:3]: I think best practice is to put dotenv it in Line 1

Ref: https://github.com/motdotla/dotenv#usage

**Review by bwnstck — APPROVED** [review]
nice one  👩‍🎤  just 2 small changes :)

File: client/src/api/connectJSON.js (hunk 1 above)
Comment (boldCM) [reply client/src/api/connectJSON.js]: thanks, good point!
Yes I actually do, for a toggle Icon that switches on true or false ;)

File: server.js (hunk 2 above)
Comment (boldCM) [reply server.js:3]: perfect, thanks, didin't know that!👍

**Pull request merged**